DEV_GUIDE.md

```diff
@@ -133,3 +133,57 @@ curl -L -X GET 'localhost:8082/v1/keyspaces' \
 --header 'content-type: application/json' \
 --header 'X-Cassandra-Token: <AUTH_TOKEN>'
 ```
+
+## Running Integration Tests
+
+Integration tests require the Cassandra Cluster Manager ((ccm)[https://github.com/riptano/ccm])
+to be installed and accessible via the OS `PATH`.
+
+Note: Integration tests use `ccm` to start transient storage nodes, that are normally destroyed at 
+the end of the test run. However, if the test JVM is killed during execution, the external storage
+node may continue running and may interfere with subsequent test executions. In this case the
+transient storage process needs to be stopped manually (e.g. by using the `kill` command).
+
+### Ordinary Execution
+
+To run integration tests in the default configuration, run:
+
+```shell
+./mvnw verify
+```
+
+This will run integration tests for Cassandra 3.11 and 4.0. 
+On a reasonably powerful laptop it takes about 40 minutes.
+
+Note: support for DSE is not turned on by default.
+To build and test Stargate with the DSE 6.8 persistence module, run:
+
+```shell
+./mvnw verify -P dse -P it-dse-6.8
+```
+
+To run integration tests with all Cassandra and DSE persistence modules, run:
+
+```shell
+./mvnw verify -P it-cassanrda-3.11 -P it-cassanrda-4.0 -P dse -P it-dse-6.8
```

Comment:
You have typo in there (s/cassanrda/cassandra/g)

Reply:
thx. fixed.